django_app/redbox_app/jinja2.py

```diff
@@ -47,6 +49,13 @@ def humanize_timedelta(minutes=0, hours_limit=200, too_large_msg=""):
         return humanize.precisedelta(delta, minimum_unit="minutes")
 
 
+def to_user_timezone(value):
```

Comment:
So, we are storing UTC in the database, and just converting to user TZ in the template? Nice.